pkg/frontend/frontend.go

```diff
@@ -256,6 +256,12 @@ func (f *frontend) authenticatedRoutes(r *mux.Router) {
 
 	s.Methods(http.MethodGet).HandlerFunc(f.getAdminOpenShiftClusters).Name("getAdminOpenShiftClusters")
 
+	s = r.
+		Path("/admin/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/{resourceProviderNamespace}/{resourceType}/{resourceName}/updatenic").
+		Subrouter()
+
+	s.Methods(http.MethodPost).HandlerFunc(f.postAdminReconcileFailedNIC).Name("updateFailedNic")
```

Comment:
can we rename `"updateFailedNic"` to `"reconcileFailedNic"`

Reply:
updated file names and endpoint for consistency